### PR TITLE
OPT-5 Endpoint for Refund Request List

### PR DESCRIPTION
**Pull request by gobliggg**
## What does this PR do?
Added new function for pagination

## Why are we doing this? Any context or related work?
_You may put your JIRA card link here_

## Where should a reviewer start?
_optional -- if your changes affected so much files, it is encouraged to give helper for reviewer_

## Screenshots
_optional -- You may put the database, sequence or any diagram needed_

## Manual testing steps?
_Steps to do tests. including all possible that can hape_

## Database changes
_optional -- If there's database changes, put it here_

## Config changes
_optional -- If there's config changes, put it here_

## Deployment instructions
_optional -- Better to put it if there's some 'special case' for deployment_

### CODE REVIEW

File: random/pagination.go [hunk 1]
```diff
@@ -0,0 +1,58 @@
+package random
```
Comment (budiariyanto) [inline random/pagination.go:1]: packagenya jgn random dongs, dijadiin satu aja ama package `http`, atau package baru di dalem `http` juga ok

Comment (gobliggg) [reply random/pagination.go]: @budiariyanto : Satuin aja lebih enak

Comment (budiariyanto) [reply random/pagination.go]: @gobliggg man, itu pagination limitnya bisa dibikin configurable nggak? sekalian

Comment (gobliggg) [reply random/pagination.go]: @budiariyanto : Pagination limitnya gak dipake mas bud, itu biar bisa dipanggil aja di app yang pake.

Comment (ardityawahyu) [reply random/pagination.go]: nah iya, buat app yg pake emang ngga butuh jadi configurable kang @gobliggg ?
Mending daripada constant begitu, ya sekalian dibuat var kosongan aja.

Comment (gobliggg) [reply random/pagination.go]: Ok berarti hapus aja ya

**Review by ardityawahyu — APPROVED** [review]

**Review by budiariyanto — APPROVED** [review]

**Pull request merged**